packaging/nnstreamer.spec

```diff
@@ -284,28 +284,23 @@ meson --buildtype=plain --prefix=%{_prefix} --sysconfdir=%{_sysconfdir} --libdir
 ninja -C build %{?_smp_mflags}
 
 export NNSTREAMER_BUILD_ROOT_PATH=$(pwd)
+export GST_PLUGIN_PATH=$(pwd)/build/gst/nnstreamer
+export NNSTREAMER_CONF=$(pwd)/build/nnstreamer-test.ini
+export NNSTREAMER_FILTERS=$(pwd)/build/ext/nnstreamer/tensor_filter
+export NNSTREAMER_DECODERS=$(pwd)/build/ext/nnstreamer/tensor_decoder
+
+%define test_script $(pwd)/packaging/run_unittests_binaries.sh
 
-pushd build
-export GST_PLUGIN_PATH=$(pwd)/gst/nnstreamer
-export NNSTREAMER_CONF=$(pwd)/nnstreamer-test.ini
-export NNSTREAMER_FILTERS=$(pwd)/ext/nnstreamer/tensor_filter
-export NNSTREAMER_DECODERS=$(pwd)/ext/nnstreamer/tensor_decoder
-%if 0%{?unit_test}
-    ./tests/unittest_common --gtest_output="xml:unittest_common.xml"
-    ./tests/unittest_sink --gst-plugin-path=. --gtest_output="xml:unittest_sink.xml"
-    ./tests/unittest_plugins --gst-plugin-path=. --gtest_output="xml:unittest_plugins.xml"
-    ./tests/unittest_src_iio --gst-plugin-path=. --gtest_output="xml:unittest_src_iio.xml"
-    ./tests/tizen_capi/unittest_tizen_capi --gst-plugin-path=. --gtest_output="xml:unittest_tizen_capi.xml"
-%endif
 %if %{with tizen}
-    %if 0%{?enable_nnfw_r}
-        ./tests/tizen_nnfw_runtime/unittest_nnfw_runtime_raw --gst-plugin-path=. --gtest_output="xml:unittest_nnfw_runtime_raw.xml"
-    %endif
+    bash %{test_script} ./tests/tizen_nnfw_runtime/unittest_nnfw_runtime_raw
     ln -s ext/nnstreamer/tensor_source/*.so .
-    ./tests/tizen_capi/unittest_tizen_sensor --gst-plugin-path=. --gtest_output="xml:unittest_tizen_sensor.xml"
+    bash %{test_script} ./tests/tizen_capi/unittest_tizen_sensor
+%endif
```

Comment:
Why these unittests are not inside `unit_test` if-statement? Was it intended?

Reply:
`unit_test` is only enabled in x86_64, and nnfw_runtime is not available for that architecture.
However, tizen sensor framework seems to be available for x86_64 architecture.
If you want to move it under `unit_test condition`, I think a different PR would be good.

Reply:
OK

Reply:
Sensor framework depends on Tizen. We want to test it unconditionally in Tizen because it may get regressions from other Tizen packages. Anyway, we have disabled most unittests in ARM because it slows down the build too much for CI systems.